.travis.yml

```diff
@@ -0,0 +1,24 @@
+notifications:
+  email:
+    on_success: never
+language: cpp
+sudo: false
+os:
+  - osx
```

Comment:
Quick question, any reason why an OS X image?

Reply:
The linux builds are done by Jenkins, and we offload the OSX builds on Travis

Reply:
Got it 😄